spec/win32_std_spec.cr

```diff
@@ -1,64 +1,75 @@
-# This file is autogenerated by `spec/generate_windows_spec.sh`
-# 2020-01-09 23:15:57+01:00
+# This file is autogenerated by `scripts/generate_windows_spec.sh`
+# 2020-04-20 22:36:41+02:00
 
-# require "./std/adler32_spec.cr" (failed linking)
+require "./std/adler32_spec.cr"
 require "./std/array_spec.cr"
 require "./std/atomic_spec.cr"
 require "./std/base64_spec.cr"
-# require "./std/benchmark_spec.cr" (failed codegen)
+require "./std/benchmark_spec.cr"
 # require "./std/big/big_decimal_spec.cr" (failed linking)
 # require "./std/big/big_float_spec.cr" (failed linking)
 # require "./std/big/big_int_spec.cr" (failed linking)
 # require "./std/big/big_rational_spec.cr" (failed linking)
-# require "./std/big/number_spec.cr" (failed codegen)
+# require "./std/big/number_spec.cr" (failed linking)
 require "./std/bit_array_spec.cr"
 require "./std/bool_spec.cr"
 require "./std/box_spec.cr"
-# require "./std/callstack_spec.cr" (failed codegen)
+# require "./std/callstack_spec.cr"
 # require "./std/channel_spec.cr" (failed codegen)
 require "./std/char/reader_spec.cr"
 require "./std/char_spec.cr"
 require "./std/class_spec.cr"
 require "./std/colorize_spec.cr"
 require "./std/comparable_spec.cr"
 require "./std/complex_spec.cr"
+require "./std/compress/deflate/deflate_spec.cr"
+require "./std/compress/gzip/gzip_spec.cr"
+require "./std/compress/zip/zip_file_spec.cr"
+require "./std/compress/zip/zip_spec.cr"
+require "./std/compress/zlib/reader_spec.cr"
+require "./std/compress/zlib/stress_spec.cr"
+require "./std/compress/zlib/writer_spec.cr"
+# require "./std/concurrent/future_spec.cr" (failed codegen)
```

Comment:
FYI this file no longer exists on master.

Reply:
Thanks. I double-checked by re-generating the list, and your two comments cover everything.